pipeline/pipeline/tasks/spreadsheets.py

```diff
@@ -106,6 +124,9 @@ def run(self):
             scrap_df = scrape_case_growth_to_df(named_tmp_file.name, page=self.page)
             scrap_df["downloaded_for"] = self.date.strftime("%Y-%m-%d")
             result_df = pandas.concat([daily_case_growth_df, scrap_df])
+
+            print(result_df)
```

Comment:
This print can get a bit big in 'production' so better to remove it or to replace it with 
import logging
log = logging.getLogger(__name__) 
and then in code
log.info(result_df)
then it can be filtered with log level's